<ui version="4.0" >
 <class>WdgColorSpaceSelector</class>
 <widget class="QWidget" name="WdgColorSpaceSelector" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>99</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="lblColorSpaces" >
     <property name="text" >
      <string>Model:</string>
     </property>
     <property name="buddy" >
      <cstring>cmbColorModels</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="KisCmbIDList" native="1" name="cmbColorModels" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lblColorModels" >
     <property name="text" >
      <string>Depth:</string>
     </property>
     <property name="buddy" >
      <cstring>cmbColorModels</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="KisCmbIDList" native="1" name="cmbColorDepth" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lblProfiles" >
     <property name="text" >
      <string>Profile:</string>
     </property>
     <property name="buddy" >
      <cstring>cmbProfile</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="SqueezedComboBox" name="cmbProfile" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisCmbIDList</class>
   <extends>QWidget</extends>
   <header>kis_cmb_idlist.h</header>
  </customwidget>
  <customwidget>
   <class>SqueezedComboBox</class>
   <extends>QComboBox</extends>
   <header>squeezedcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
